src/plugins/requestAuthPlugin.js

```diff
@@ -107,17 +107,17 @@ export const applyRequestAuthorizer$ = requestAuthorizer$ => request => {
  * Get an anonymous code from the API that can be used to generate an oauth
  * access token
  *
- * @param {Object} config { OAUTH_AUTH_URL, oauth }
+ * @param {Object} server the Hapi app server instance
```

Comment:
oooh, I haven't come across `server.settings.app` - that's definitely what we should be using. We'll get into long property chains when accessing it, but that's probably okay

```js
const ouathKey = response.request.server.settings.app.oauth.key;
```